### ISSUE TITLE
GPSLogger has been removed from the Play Store, and development has been paused.

### ISSUE THREAD

**Issue by mendhak** [mendhak/gpslogger#849]
Just yesterday GPSLogger has again been unpublished from the Play Store (for the _nth_ time).  Play Store removals are tricky business; I often don't see much reason given except vague statements and links to policy documents, and therefore these situations often require a lot of guessing, rework and resubmitting.  These events are a highly stressful time as it's a race to quickly fix things and get the app back on the store.

In the past I've spent effort in getting the app reinstated, however I can't do that any longer; I no longer have the motivation and energy to support GPSLogger development and to deal with the Play Store's moving goalposts.  My attention has been focused elsewhere over the past year, and this is evident by my slow replies lack of attention to issues and emails (sorry!).  More importantly I am not willing to endure the stress that these store removals cause me.   

The core of the issue (why it gets removed) has been that GPSLogger supports a lot of _older_ devices and frameworks, whereas the Play Store is always trying to get everyone onto the latest versions.  It's not obvious from here but a majority of GPSLogger users are on older devices.  This has led to a lot of complexity in the codebase and inevitably (and expectedly) made it difficult to support and maintain while keeping up with the latest Play Store requirements and user feature requests and PRs and issues, and giving them the attention they deserve.  

I'll keep trying to answer some Github issues and direct emails when I can, but still at a slow pace. The latest APK is [still available in releases section](https://github.com/mendhak/gpslogger/releases) but will now require sideloading. 

Future options are unknown, I've been trying to mull these over... I don't want to hand the development, account and app over to potential future maintainers; in the unlikely event of someone else wanting to fork and republish, I'd rather the fork get republished with a different name.  I say unlikely event, because as mentioned before, the codebase is complex and it might be easier for someone new to just start over.  If the motivation and energy ever return I could consider moving to normal F-Droid, which will require stripping out GDrive, Dropbox and a few other proprietary features.  

tl;dr - Removed from Play Store, reason unknown. Low energy, high stress. Development on hold.

**Comment by GrazianoCapelli** [comment]
@mendhak First of all, thanks for sharing your precious work with the Community.
I strongly hope you'll recharge your battery, regain motivation, and come back developing and maintaining this damn good open source software.

**Comment by NelsonMinar** [comment]
Thank you so much for your work over the years; GPSLogger has been an excellent and very flexible tool. I hope you can find some future for it, either yourself or with a new maintainer. Certainly understood getting fed up with it all though and if this is the end, well, thank you so much!

**Comment by sm2x** [comment]
I hope you will reconsider, our business is GPS-based and we always trust your App as the defacto standard when it comes to reliability, more power to you mendhak!

**Comment by Debianer** [comment]
GPSLogger has been my trusted companion through the years, from the first Android I owned, to today. I had different uses for it. Publishing routes, as a client for Nextcloud PhoneTrack and tagging pictures. It was always installed and lately, because of Nextcloud  PhoneTrack, always running. While I am sad to see it go, I understand and support Menhac in this decision. 

It would be nice if the app could reappear on the Play Store, perhaps explicitly stating it won't support newer Android version but supporting older Android versions, as you correctly state that those aren't supported by many GPS Loggers. Many users don't know how to sideload an app. And those also often use older phones.

**Comment by joergRossdeutscher** [comment]
Thanks a lot for your excellent work.

**Comment by nickw1** [comment]
@mendhak only just seen this. Do not understand why it has been removed from the Play Store, it's an open source GPS tracking app; not some privacy-violating app which surreptitiously sends your data to some unnamed party. It has been my prime GPS surveying app for many years so many thanks for creating it. Thank you for continuing to make APKs available.

**Comment by mendhak** [comment]
Just an update, for convenience I've made the latest version of the app available on normal F-Droid:  https://f-droid.org/en/packages/com.mendhak.gpslogger/

Google Drive integration and Activity recognition have been removed as they make use of Play Services libraries, which is not allowed on F-Droid.  

I'll also update the top comment in this thread.

**Comment by rekcodocker** [comment]
I just found it at F-Droid and can I just say THANK YOU. 
I have tried various apps while GPS Logger was unavailable.
None come even close to the upload capabilities and the subtle balance between accurate logging and battery optimization.
Thank you!

**Comment by theAkito** [comment]
No stress, buddy. I know exactly how you feel.

**Comment by mehrdadn** [comment]
I just saw this thread and thought I'd make a suggestion here: it may be worth considering making a _paid_ Google Play version, while keeping the GitHub and F-Droid versions free. The reason being that many phones (Intune, corporate, etc.) simply do not allow installation from outside the Play Store, so there are people who cannot use your app no matter how much they want to—even those who are willing to pay for it. And of course the income might help with motivation on your side as well—especially if you make it a recurring subscription (e.g., $1/month) rather than a one-time payment.

Of course money can't substitute for lower stress or higher energy, so perhaps this wouldn't make sense in your case. But I thought it might be worth mentioning in case it might be useful depending on your situation.

Thanks for making the app!
